Tests/FluentAssertions.Specs/Specialized/TaskCompletionSourceAssertionSpecs.cs

```diff
@@ -73,6 +73,38 @@ public async Task When_it_completes_in_time_and_it_is_not_expected_it_should_fai
             await action.Should().ThrowAsync<XunitException>().WithMessage("*to not complete within*because test testArg*");
         }
 
+        [Fact]
+        public async Task When_it_is_canceled_before_completion_and_it_is_not_expected_it_should_fail()
```

Comment:
The inferior naming is due to `CompleteWithinAsync` and `NotCompleteWithinAsync` are in the same nested class.